### DISCUSSION TITLE
'FirestoreFieldPath' cannot be used in conjunction with a different Query.orderBy() parameter error

### DISCUSSION THREAD

**Discussion by V3RT1AG0** [invertase/react-native-firebase#4966] [Q&A]
Error: firebase.firestore().collection().orderBy() Invalid query. Query.where() fieldPath parameter: 'FirestoreFieldPath' cannot be used in conjunction with a different Query.orderBy() parameter

I am getting above error even when I am using FirestoreFieldPath as first orderBy parameter. As mentioned [here](https://github.com/googleapis/nodejs-firestore/issues/990#issuecomment-754737700) it should work. Below is my block of code.

```
globalGroups
          .where(firestore.FieldPath.documentId(), 'in', conversationIdArr)
          .orderBy(firestore.FieldPath.documentId())
          .orderBy('updatedAt') //no error after removing this line 
          .get()
```

**Comment by mikehardy** [comment]
Hi there! If you are reporting an issue you should report an issue (following the template, which provides necessary details to offer help) vs starting a discussion
